futurenhs.content.api/Umbraco9ContentApi.Core/Services/FutureNhs/FutureNhsContentService.cs

```diff
@@ -79,9 +82,13 @@ public IContent CreateContent(string name, Guid parentId, string documentTypeAli
         public void DeleteContent(Guid contentId, CancellationToken cancellationToken)
         {
             var content = _contentService.GetById(contentId);
-
-            if (!_contentService.Delete(content).Success)
-                throw new KeyNotFoundException($"Unable to delete content {contentId}. Content does not exist.");
+            if (content is not null)
+            {
+                if (!_contentService.Delete(content).Success)
+                {
+                    _logger.LogWarning("Unable to delete content {0}. Content does not exist.", contentId);
```

Comment:
sorry to be super pedantic but could the property name better describe parameter